src/encoding.ts

```diff
@@ -283,6 +283,11 @@ export interface Encoding<F extends Field> {
    */
   url?: StringFieldDefWithCondition<F> | StringValueOrSignalDefWithCondition<F>;
 
+  /**
+   * A string that labels the axis group for accessibility.
+   */
+  ariaLabel?: StringFieldDefWithCondition<F> | StringValueOrSignalWithCondition<F> | StringFieldDef<F>[];
```

Comment:
I thought we said that the initialize version doesn't need array?

Reply:
I forgot to change it back. Fixed.